catalog/edf-methanesat-ee/projects_edf-methanesat-ee_assets_public-preview_L3concentration.jsonnet

```diff
@@ -0,0 +1,186 @@
+  description: |||
+    This early "Public Preview" dataset provides geospatial data for the 
+    column-averaged dry-air mole fraction of methane in the atmosphere, "XCH4", 
+    retrieved from measurements by the MethaneSAT imaging spectrometer. XCH4 is 
+    defined as the total column amount (number of molecules above a unit surface 
+    area) of methane ("CH4") divided by the total amount of air (number of molecules 
+    above a unit surface area, water vapor amount removed) along the line of sight 
+    from the satellite’s spectrometer to the Earth's surface and back up to the Sun. 
+    Additional data layers are provided for reference: observed albedo at 1622 nm, 
+    surface pressure, and terrain height.
+
+    The spatial footprint of individual MethaneSAT soundings is 110 m x 400 m when 
+    viewing at nadir, with the long side in the satellite’s flight direction, and 
+    with the short side that can be stretched wider at higher viewing zenith angles 
+    (which is why some targets appear more rectangular than others). At nadir, the 
+    swath width is ~220 km. The data on these native soundings is Level 2 (L2) data. 
+    The precision of the L2 XCH4 is 22-35 ppb (corresponding to 0.7-0.2 surface albedo).
+    L2 concentrations (XCH4) are retrieved from the measured high resolution (0.08 nm 
+    sampling, 0.24 nm resolution) spectra from the 1.6 micron band using the CO2-Proxy 
+    method ([Chan Miller et al. (2024)](https://doi.org/10.5194/amt-17-5429-2024)). 
+    The Level 3 (L3) dataset is created by regridding the L2 data from the native 
+    sounding locations to a regular 45 m x 45 m grid using the approach detailed in 
+    [Sun et al. (2018)](https://doi.org/10.5194/amt-11-6679-2018). 
+    
+    Contact the data provider for more information about the project at this link: 
+    [https://www.methanesat.org/contact/](https://www.methanesat.org/contact/).
+  |||,
+  keywords: [
+    'climate',
+    'emissions',
+    'ghg',
+    'methane',
+    'edf',
+    'methanesat',
+  ],
+  providers: [
+    ee.producer_provider('Environmental Defense Fund - MethaneSAT', 'https://methanesat.org'),
+    // This is always the last entry.
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2006-01-24T00:00:00Z', '2025-01-17T00:00:00Z'),
+  summaries: {
+    'gsd': [10.2],
+    'eo:bands': [
+      {
+        name: 'XCH4',
+        description: 'Retrieved column-averaged dry-air CH4 mole fraction.',
+        'gee:units': units.ppb,
+      },
+      {
+        name: 'albedo',
+        description: 'Ratio of solar radiance at the ground to observed ' +
+        'radiance at the sensor.',
+      },
+      {
+        name: 'surface_pressure',
+        description: 'Air pressure at the surface.',
+        'gee:units': units.hectopascal,
+      },
+      {
+        name: 'terrain_height',
+        description: 'Elevation with respect to WGS84 reference ellipsoid.',
+        'gee:units': units.kilometer,
```

Comment:
DEM datasets typically use meters, but I guess this is too late to change.

Reply:
Thanks.  I verified that it is meters and updated this.